lib/jsdom/living/svg/SVGPreserveAspectRatio-impl.js

```diff
@@ -0,0 +1,98 @@
+"use strict";
+
+const DOMException = require("domexception/webidl2js-wrapper");
+
+const alignmentToStringMap = [
```

Comment:
Hmm, although I see upon reading the code that this is a "map" from the numbers 0 - 10, it's a bit too clever. Some options:

- Add a comment explaining
- Change the name, maybe `alignmentStringsByIndex` or something
- Compute the inverse of `alignmentToIntMap` at runtime, making this `const alignmentToStringMap = inverse(alignmentToIntMap)`.
- Just spell it out by using an object literal with keys 0-10.